unittests/QSharedPointer_include/test_qsharedpointer_include.cpp

```diff
@@ -0,0 +1,20 @@
+#include "wsdl_test_qsharedpointer_include_wsdl.h"
+
+#include <QtTest>
+
+class rightInclude:
+  public QObject
+{
+Q_OBJECT
+
+private Q_SLOTS:
+
+  void testCompiled()
+  {
+    QVERIFY(true);
```

Comment:
This statement does nothing, please remove it.

You could at least create an instance of the service, so that this doesn't compile with a fully empty generated file ;)

    Hello_Service service;